src/ui/SearchAlerts/SearchAlerts.ts

```diff
@@ -322,26 +322,21 @@ export class SearchAlerts extends Component {
     table.append(tableBodySubscriptions.el);
     let sizeModForModalBox = 'big';
 
-    return this.queryController
-      .getEndpoint()
-      .listSubscriptions()
-      .then((subscriptions: ISubscription[]) => {
-        _.each(subscriptions, subscription => {
-          this.addSearchAlert(subscription, container);
-        });
-      })
-      .catch(() => {
-        sizeModForModalBox = 'small';
-        container.empty();
-        container.append(this.getFailureMessage().el);
-      })
-      .finally(() => {
-        this.modal = this.ModalBox.open(container.el, {
-          title: title.el.outerHTML,
-          className: 'coveo-subscriptions-panel',
-          sizeMod: sizeModForModalBox
-        });
-      });
+    try {
+      const subscriptions = await this.queryController.getEndpoint().listSubscriptions();
+      _.each(subscriptions, subscription => this.addSearchAlert(subscription, container));
```

Comment:
subscriptions.forEach[...]

Reply:
I'm going to postpone this change for the same reason as above where underscore handles falsy values.